nativescript-charter.md

```diff
@@ -0,0 +1,105 @@
+
+## Section 1. Scope
+
+NativeScript brings native platform APIs directly into the JavaScript runtime, enriched with robust TypeScript typings, ensuring a seamless and powerful development experience. As an open-source framework for building applications across iOS, visionOS, Android, macOS, and more, it uniquely blends familiar web development paradigms—such as CSS styling and template-driven views—with popular native languages including Swift, Kotlin, Objective-C, and Java. This integration offers developers unparalleled flexibility and ease.
+Its intuitive CLI further simplifies interaction with native platform development processes, making native development approachable for JavaScript developers.
+
+### 1.1: In-scope
+
+Section Intentionally Left Blank
+
+### 1.2: Out-of-Scope
+
+Section Intentionally Left Blank
+
+## Section 2. Relationship with OpenJS Foundation CPC.
+
+Most large, complex open source communities have both a business and a technical governance model. Technical leadership for the projects within the OpenJS Foundation is delegated to the projects through their project charters by the OpenJS Cross Project Council (CPC). In the case of NativeScript, it is delegated to the NativeScript Technical Steering Committee ("TSC"). OpenJS Foundation's business leadership is the Board of Directors (the "Board").
+
+This charter can only be amended with the approval of the CPC.
+
+## Section 3. Establishment of the TSC
+
+TSC memberships are not time-limited. There is no maximum size of the TSC.
+
+The TSC may add additional members to the TSC through meeting consensus. The consensus requires at least five TSC members to be present to approve a new member.  A TSC member can be removed from the TSC by voluntary resignation or by consensus with at least five members present.
+
+No more than two-thirds of the TSC members may be affiliated with the same employer. If a change in TSC membership or a change of employment by a TSC member creates a situation where more than two-thirds of the TSC membership shares an employer, then the situation must be immediately remedied by the removal of a member affiliated with the over-represented employer(s).
+
+The TSC shall meet regularly using tools that enable participation by the community (e.g. weekly on a Zoom meeting, or through any other appropriate means selected by the TSC). The meeting shall be directed by the TSC Chairperson. Responsibility for directing individual meetings may be delegated by the TSC Chairperson to any other TSC member. Minutes or an appropriate recording shall be taken and made available to the community through accessible public postings.
+
+TSC members are expected to regularly participate in TSC meetings and issue triaging.
+
+A TSC member is automatically removed if they do not participate in at least 2 annual meetings.
```

Comment:
- "annual meetings" are not defined in this doc.  is it defined somewhere else?
- "automatic" removal of members can be contentious.  consider revising